### ISSUE TITLE
Copy/Paste from Attachment

### ISSUE THREAD

**Issue by solepixel** [Thomas101/wmail#356]
This is probably a low priority, but thought I'd add this in here. There doesn't appear to be a way to copy/paste when opening attachment in Gmail's "Preview Mode". If I click on an attachment, Gmail will open the document (in this case .rtf) in a modal window. From within this window, I can select text but cannot copy. If I try to right-click, there is only Undo, Redo, Select All, WMail Settings and Inspect, and the first 3 options are disabled. Copy/paste works fine from the browser, just not inside of WMail (when previewing attached text files).

* Which version of WMail are you using? 1.4.0 prerelease
* Which Operating System are you using? Mac OSX 10.12 (Sierra)

**Comment by Thomas101** [comment]
I think this could be intentional on Googles part. I've just tried it in chrome and firefox and see the same thing. I can even highlight the text! Copying is out of the question though. I tried right click and `cmd+c`.

I had a look to see what was happening in the dom. It looks like they render the rtf then place an image version of the rtf document over it. Then if you try removing some of the dom elements over the text they end up getting added back in and all sorts.

Looks like this has been around [since 2014 too](https://discussions.apple.com/thread/6720329?start=0&tstart=0)
